zh/backup-to-gcs.md

```diff
@@ -1,11 +1,12 @@
 ---
-title: 使用 Mydumper 备份 TiDB 集群数据到 GCS
-summary: 介绍如何使用 Mydumper 将 TiDB 集群数据备份到 Google Cloud Storage (GCS)。
+title: 使用 Dumpling 备份 TiDB 集群数据到 GCS
+summary: 介绍如何使用 Dumpling 将 TiDB 集群数据备份到 Google Cloud Storage (GCS)。
+category: how-to
 ---
 
-# 使用 Mydumper 备份 TiDB 集群数据到 GCS
+# 使用 Dumpling 备份 TiDB 集群数据到 GCS
 
-本文档详细描述了如何将 Kubernetes 上 TiDB 集群的数据备份到 [Google Cloud Storage (GCS)](https://cloud.google.com/storage/docs/) 上。本文档中的“备份”，均是指全量备份（Ad-hoc 全量备份和定时全量备份），底层通过使用 [`mydumper`](https://pingcap.com/docs-cn/v3.0/reference/tools/mydumper) 获取集群的逻辑备份，然后再将备份数据上传到远端 GCS。
+本文档详细描述了如何将 Kubernetes 上 TiDB 集群的数据备份到 [Google Cloud Storage (GCS)](https://cloud.google.com/storage/docs/) 上。本文档中的“备份”，均是指全量备份（Ad-hoc 全量备份和定时全量备份），底层通过使用 [`Dumpling`](https://pingcap.com/docs-cn/stable/dumpling-overview/) 获取集群的逻辑备份，然后再将备份数据上传到远端 GCS。
```

Comment:
```suggestion
本文档详细描述了如何将 Kubernetes 上 TiDB 集群的数据备份到 [Google Cloud Storage (GCS)](https://cloud.google.com/storage/docs/) 上。本文档中的“备份”，均是指全量备份（Ad-hoc 全量备份和定时全量备份），底层通过使用 [`Dumpling`](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview) 获取集群的逻辑备份，然后再将备份数据上传到远端 GCS。
```
Recommend linking to the new doc site so that when the old site goes completely offline, we don't have to update it again.